scripts/porto_seguro/preprocess_porto_seguro.py

```diff
@@ -0,0 +1,134 @@
+import os
+import json
+import logging
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+logging.basicConfig(level=logging.INFO)
+
+RAW_DATASET_PATH = os.path.realpath("../../sdgym/data/porto-seguro/train.csv.zip")
+NP_DATASET_PATH = os.path.realpath("../../sdgym/data/porto-seguro.npz")
+JSON_FILE_PATH = os.path.realpath("../../sdgym/data/porto-seguro.json")
+
+def create_ordinal_column(data_df, col_name, label_col_name="target"):
+
+    i2s = [str(val) for val in sorted(data_df[col_name].unique())]
+    col_size = len(i2s)
+    col_type = "ordinal"
+    if col_name == label_col_name:
+        col_name = "label"
+
+    column_object = {
+        "i2s": i2s,
+        "name": col_name,
+        "size": col_size,
+        "type": col_type
+    }
+    return column_object
+
+def create_categorical_column(data_df, col_name, label_col_name="target"):
+
+    i2s = [str(val) for val in sorted(data_df[col_name].unique())]
+    col_size = len(i2s)
+    col_type = "categorical"
+    if col_name == label_col_name:
+        col_name = "label"
+
+    column_object = {
+        "i2s": i2s,
+        "name": col_name,
+        "size": col_size,
+        "type": col_type
+    }
+    return column_object
```

Comment:
Those two functions are exactly the same beside the `col_type`. Could be an agument instead.